circle.yml

```diff
@@ -14,17 +14,32 @@ machine:
     PATH: $PATH:$HOME/.local/bin
     CLOUDSDK_CORE_DISABLE_PROMPTS: 1
     STATE: /home/ubuntu/state.env
+    NUM_HOSTS: 16
```

Comment:
To recap what has been discussed offline:

1. we used to have 10 cores, however
2. I noticed that by increasing to 16 cores, it would parallelise tests further and decrease the build time by ~1-2 minutes.
3. increasing to more than 16 cores wouldn't change much, as we would be waiting for some of the long-running tests.

This obviously assumes we value developer time & timely feedback more than increased costs for testing infrastructure.